### PR TITLE
[v4.2] Cirrus: Improve CI VM image updates for EC2

### PR DESCRIPTION
**Pull request by cevich**
AWS EC2 keys VM images by an utterly unreadable, horrible to use,
generated "AMI ID" value.  This is very error prone for humans in
practice, since it's impossible to tell one image from the next by
eye.  Worse, EC2 permits duplicate name-tag values, complicating
image specification further.

However fortunately, Cirrus-CI recently implemented a feature by
which AMI's may be referenced by a name-tag search - choosing
the most recent AMI found.  Since the `containers/automation_images`
build workflow always assigns a unique name + `$IMAGE_SUFFIX` value,
we can simply re-use it for both AWS and GCP image specification.

In other words as of this commit, specifying new CI VM images can
be done by simply updating the `$IMAGE_SUFFIX` value as we've always
done.  No need to call out a specific AMI ID just for EC2 tasks.

Signed-off-by: Chris Evich <cevich@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

**Comment by cevich** [comment]
`Building: 4bb532c45 Vendor in containers/(storage,image, common, buildah)` is blowing up the binary-size bloat check :cry:

**Comment by cevich** [comment]
Force-push: Added commit to use `$DEST_BRANCH=v4.2`.  Maybe this will make the bloat-check happy.

**Review by vrothberg — APPROVED** [review]
LGTM
@containers/podman-maintainers
